randovania/games/super_metroid/json_data/Crateria.txt

```diff
@@ -44,24 +44,29 @@ Extra - save_index: 0
 > Landing Site Save Station; Heals? True; Spawn Point; Default Node
   * Layers: default
   > Door to Gauntlet Access
-      All of the following:
+      Any of the following:
```

Comment:
This conversion was likely in error. Otherwise, it would imply that just wall jumps with no items can get you into gauntlet.

Reply:
That was indeed an error.

Reply:
Ok, then fix it when you get the chance.